bindings/SofaTypes/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ SP3_add_python_package(
         SofaTypes
 )
 
-find_package(SofaFramework REQUIRED)
+find_package(Sofa.DefaultType REQUIRED)
```

Comment:
[Bindings.SofaTypesConfig.cmake.in](https://github.com/sofa-framework/SofaPython3/blob/master/bindings/SofaTypes/Bindings.SofaTypesConfig.cmake.in) is not up to date.